releasenotes/notes/unify-instructions-and-commands-aaa6d8724b8a29d3.yaml

```diff
@@ -0,0 +1,22 @@
+---
+prelude: >
```

Comment:
This should probably be either `feature` or `upgrade` (probably feature)

Reply:
My idea was this one prelude message and then the following 6 PRs would be a list of upgrades with specific details, for example DelayInstruction(Delay(time), channel) => Delay(time, channel)

Reply:
But I'll defer to your opinion